cost/aws/idle_compute_instances/idle_compute_instances.pt

```diff
@@ -713,17 +726,20 @@ script "js_instance_cost_mapping", type:"javascript" do
           "mem_average": instance["mem_average"],
           "mem_maximum_value": instance["mem_maximum_value"],
           "mem_average_value": instance["mem_average_value"]
-          "savings": cur + ' '+formatNumber(monthly_savings.toFixed(3), separator)
+          "savings": parseFloat(monthly_savings.toFixed(3)),
```

Comment:
Rather than rendering to a string and then parsing it back to a number, it would probably be better to do something like:

```javascript
Math.round(monthly_savings * 1000) / 1000;
```